README.md

```diff
@@ -1 +1,65 @@
-#  Chart repo
+# OpenShift Helm Charts
+
+OpenShift Helm Charts is a repository hosting [Helm Charts](https://github.com/helm/helm) available by default with [OpenShift](https://www.openshift.com/). It contains popular technologies, tools and services. Helm Charts on this repository can be provided by the community, by partners or Red Hat. 
+
+Charts optionaly go through an automated RedHat OpenShift certification workflow, which garanties security compliance as well as best integration and experience with the platform.
```

Comment:
is it really optional?

Reply:
Good question, if we provide community path, it seems it is. You tell me

Reply:
community helm charts are not supposed to go through the certification process.